### PR TITLE
Adds Director of ML Insights Part 2

### PR DESCRIPTION
**Pull request by BritneyMuller**
Screen Shot 2022-05-05 at 4 36 01 PM

screencapture-localhost-5564-blog-ml-director-insights-2-2022-05-05-16_54_49

### CODE REVIEW

File: ml-director-insights-2.md [hunk 1]
```diff
@@ -0,0 +1,219 @@
...
+            <span class="fullname">Britney Muller</span>
+        </div>
+    </a>
+</div>
+</head>
+<body>
+
+Welcome to Part 2 of our [Director of Machine Learning Insights Series](https://huggingface.co/blog/ml-director-insights).
+
+Directors of Machine Learning have a unique seat at the AI table spanning the perspective of various roles and responsibilities. Their rich knowledge of ML frameworks, engineering, architecture, real-world applications and problem-solving provides deep insights into the current state of ML. For example, one Director will note how using new transformers speech technology decreased their team’s error rate by 30% and how simple thinking can help save _a lot_ of computational power.
```
Comment (feconroses) [inline ml-director-insights-2.md:33]:
```suggestion
Directors of Machine Learning have a unique seat at the AI table spanning the perspective of various roles and responsibilities. Their rich knowledge of ML frameworks, engineering, architecture, real-world applications and problem-solving provides deep insights into the current state of ML. For example, one director will note how using new transformers speech technology decreased their team’s error rate by 30% and how simple thinking can help save _a lot_ of computational power.
```

File: ml-director-insights-2.md [hunk 2]
```diff
@@ -0,0 +1,219 @@
...
+
+In this second SaaS focused installment, you’ll hear from a deep learning for healthcare Author who founded a non-profit for mentoring ML talent, a chess fanatic Cybersecurity expert, an Entrepreneur whose business was inspired by Barbie’s need to monitor brand reputation after a lead recall, and a seasoned patent and academic paper author who enjoys watching his 4 kids make the same mistakes as his ML models.
```
Comment (feconroses) [inline ml-director-insights-2.md:35]:
```suggestion
In this second SaaS focused installment, you’ll hear from a deep learning for healthcare textbook author who also founded a non-profit for mentoring ML talent, a chess fanatic cybersecurity expert, an entrepreneur whose business was inspired by Barbie’s need to monitor brand reputation after a lead recall, and a seasoned patent and academic paper author who enjoys watching his 4 kids make the same mistakes as his ML models.  
```

Comment (feconroses) [reply ml-director-insights-2.md:35]: I would lowercase these words here (seems a bit odd to me that these words are capitalized).

Comment (feconroses) [reply ml-director-insights-2.md:35]: also wondering to just say _"deep learning for healthcare textbook author"_ and remove _"who also founded a non-profit for mentoring ML talent"_ to keep it simple / easier to read

File: ml-director-insights-2.md [hunk 3]
```diff
@@ -0,0 +1,219 @@
...
+**Background:** Omar leads a team of Machine Learning and Data Engineers in leveraging ML for defensive security purposes as part of the Cybersecurity team. Previously, Omar has led data science and machine learning engineering teams at Adobe and SAP focusing on bringing intelligent capabilities to marketing cloud and procurement applications. Omar holds a Master’s degree in Electrical Engineering from Arizona State University. 
+
+**Fun Fact:** Omar loves to play chess and volunteers his free time to guide and mentor graduate students in AI.
+
+**Salesforce:** World's #1 customer relationship management software.
+ 
+#### **1. How has ML made a positive impact on SaaS?**
+ML has benefited SaaS offerings in many ways. Here are a few examples:
+
+a. Improving automation within applications: Example: A service ticket router using NLP to understand the context of the service request and routing it to the appropriate team within the organization.
```
Comment (feconroses) [inline ml-director-insights-2.md:52]: 😍 love this use case! It was one of the main use cases at MonkeyLearn and it changed (for good) how customer service teams work

**Review by feconroses — APPROVED** [review]
@BritneyMuller great job with this piece! Some general thoughts/suggestions:

- Can you add a CTA at the intro? I would try a "written" CTA (vs image). I feel that if we only include a CTA at the end of the post, the CTR to one of our solutions will be very low.
- I think we can optimize Martin's image?
- I'm wondering if, in the intro, we should mention some of the companies these directors work? We have some big names companies here (mainly Salesforce but ZoomInfo and to a lesser extent Amplitude are well known too) and mentioning these companies straight away in the intro may invite the user to keep reading? 
- **Absolutely loved Raphael's responses**. I'm thinking that maybe we could put his responses at the #2 spot (after Salesforce?). 
- Also, maybe we can do a particular guest blog post with Raphael covering more in-depth their speech-to-text + classification use case. 
- I liked Martin's responses too; I feel that he shared a bit of his unique insights from his experience at Repunstate.  Maybe we can put him #3 and Cao last?
- Having said this, maybe I would remove Martin's response to question #5? Personally, I find it a bit generic/uninteresting?
- To be honest, Omar's and Cao's responses were a bit underwhelming. I would have loved to learn some insider insights on how ML is being used on SFDC or Amplitude or know more about the specific ML challenges they are facing in these companies. Would it be possible to ask them to expand a bit more on 1-2 questions? Maybe questions 2-3? If not, let's try to ask directors to expand a bit more in future installments of this series.

**Comment by BritneyMuller** [comment]
> * Can you add a CTA at the intro? I would try a "written" CTA (vs image). I feel that if we only include a CTA at the end of the post, the CTR to one of our solutions will be very low.
Done!
> * I think we can optimize Martin's image?
Done!
> * I'm wondering if, in the intro, we should mention some of the companies these directors work? We have some big names companies here (mainly Salesforce but ZoomInfo and to a lesser extent Amplitude are well known too) and mentioning these companies straight away in the intro may invite the user to keep reading?
Good idea.
> * **Absolutely loved Raphael's responses**. I'm thinking that maybe we could put his responses at the #2 spot (after Salesforce?).
Thank you! Worked really hard writing those after a long video conversation with Raphael 😅 
> * Also, maybe we can do a particular guest blog post with Raphael covering more in-depth their speech-to-text + classification use case.
He would likely be open to that
> * I liked Martin's responses too; I feel that he shared a bit of his unique insights from his experience at Repunstate.  Maybe we can put him #3 and Cao last?
Representation matters and want to have Cao second.
> * Having said this, maybe I would remove Martin's response to question #5? Personally, I find it a bit generic/uninteresting?
Done!
> * To be honest, Omar's and Cao's responses were a bit underwhelming. I would have loved to learn some insider insights on how ML is being used on SFDC or Amplitude or know more about the specific ML challenges they are facing in these companies. Would it be possible to ask them to expand a bit more on 1-2 questions? Maybe questions 2-3? If not, let's try to ask directors to expand a bit more in future installments of this series.
Tried doing that before seeing this. :) They didn't provide much additional value. Added a few things but not much. Will keep this in mind for future installments.

**Comment by BritneyMuller** [comment]
@feconroses let me know if these updates look okay to you. Thanks!

File: ml-director-insights-2.md [hunk 4]
```diff
@@ -0,0 +1,219 @@
...
+<body>
+
+_If you or your team are interested in building ML solutions faster visit [hf.co/support](https://huggingface.co/support?utm_source=article&utm_medium=blog&utm_campaign=ml_director_insights_2) today!_
+
+
+👋 Welcome to Part 2 of our Director of Machine Learning Insights [Series]. Checkout [Part 1 here.](https://huggingface.co/blog/ml-director-insights)
+
+Directors of Machine Learning have a unique seat at the AI table spanning the perspective of various roles and responsibilities. Their rich knowledge of ML frameworks, engineering, architecture, real-world applications and problem-solving provides deep insights into the current state of ML. For example, one director will note how using new transformers speech technology decreased their team’s error rate by 30% and how simple thinking can help save _a lot_ of computational power.
+
+Ever wonder what Directors at Salesforce or ZoomInfo currently think about the state of Machine Learning? What their biggest challenges are? And what they're most excited about? Well, you're about to find out!
```
Comment (feconroses) [inline ml-director-insights-2.md:38]:
```suggestion
Ever wonder what directors at Salesforce or ZoomInfo currently think about the state of Machine Learning? What their biggest challenges are? And what they're most excited about? Well, you're about to find out!
```

**Review by feconroses — APPROVED** [review]

**Comment by feconroses** [comment]
@BritneyMuller lgtm!

File: ml-director-insights-2.md [hunk 5]
```diff
@@ -0,0 +1,219 @@
...
+        </div>
+    </a>
+</div>
+</head>
+<body>
+
+_If you or your team are interested in building ML solutions faster visit [hf.co/support](https://huggingface.co/support?utm_source=article&utm_medium=blog&utm_campaign=ml_director_insights_2) today!_
+
+
+👋 Welcome to Part 2 of our Director of Machine Learning Insights [Series]. Checkout [Part 1 here.](https://huggingface.co/blog/ml-director-insights)
```
Comment (osanseviero) [inline ml-director-insights-2.md:34]:
```suggestion
👋 Welcome to Part 2 of our Director of Machine Learning Insights [Series]. Check out [Part 1 here.](https://huggingface.co/blog/ml-director-insights)
```

File: ml-director-insights-2.md [hunk 6]
```diff
@@ -0,0 +1,219 @@
...
+**Background:** Omar leads a team of Machine Learning and Data Engineers in leveraging ML for defensive security purposes as part of the Cybersecurity team. Previously, Omar has led data science and machine learning engineering teams at Adobe and SAP focusing on bringing intelligent capabilities to marketing cloud and procurement applications. Omar holds a Master’s degree in Electrical Engineering from Arizona State University. 
+
+**Fun Fact:** Omar loves to play chess and volunteers his free time to guide and mentor graduate students in AI.
+
+**Salesforce:** World's #1 customer relationship management software.
+ 
+#### **1. How has ML made a positive impact on SaaS?**
+ML has benefited SaaS offerings in many ways. 
+
+**a. Improving automation within applications:** For example, a service ticket router using NLP to understand the context of the service request and routing it to the appropriate team within the organization.
```
Comment (osanseviero) [inline ml-director-insights-2.md:57]: Given it's first mention of NLP, maybe we should put the whole name

**Review by osanseviero — APPROVED** [review]
I didn't do a deep review, but it looks like a very nice blog post :+1: nice work

**Pull request merged**